Program.cs

```diff
@@ -1,6 +1,74 @@
 ﻿using System;
 using CodingCampusCSharpHomework;
 
+class Storage
+{
+    public bool LoadFromString(string descriptionString)
+    {
+        string[] fields = descriptionString.Split(';');
+        bool isParseSucceed = fields.Length == 4;
```

Comment:
No use to initialize this var if you're not gonna check it anyway before proceeding.